### ISSUE TITLE
Are there any sources that do not use vue.js but are written in the old style of HTML and Javascript?

### ISSUE THREAD

**Issue by mapinusRND** [vagran/dxf-viewer-example-src#19]
I have a question because the project I want to apply it to uses the old HTML and Javascript methods.

**Comment by vagran** [comment]
No. You can use this example as reference to make it in plain old JS. Also you most probably will need to make a bundle to link with your HTML, using a tool like rollup, webpack or vite.

**Comment by mapinusRND** [comment]
As you suggested, I solved the problem using webpack. Thank you.

**Comment by cimslab** [comment]
mapinusRND, do you mind sharing your solution, I also want a simple javascript + html implementation. Thanks!
